grafana/grafana.libsonnet

```diff
@@ -224,6 +224,10 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       deployment.mixin.metadata.withNamespace($._config.namespace) +
       deployment.mixin.metadata.withLabels(podLabels) +
       deployment.mixin.spec.selector.withMatchLabels(podLabels) +
+      deployment.mixin.spec.template.metadata.withAnnotations({
+        'checksum/grafana-config': std.md5(std.toString($.grafana.config)),
```

Comment:
I believe you meant `$._config.grafana.config`?

Reply:
Sorry, I hadn't checked the CI and hadn't noticed the condition on the config, but I do mean to hash the ConfigMap, since it contains everything including the LDAP config.

Reply:
ah yes makes sense!